### DISCUSSION TITLE
This project helped with my motivation

### DISCUSSION THREAD

**Discussion by MarkLee7916** [benawad/dogehouse#2861] [General]
I'd like to thank Ben Awad for this project and all the dogelogs, which have helped motivate to keep building projects

**Comment by nadirabbas** [comment]
ben probably didnt read this

**Reply by S-codes14** [reply]
Or this

**Reply by nadirabbas** [reply]
or this

**Reply by benawad** [reply]
or this

**Reply by nadirabbas** [reply]
I think he read this one

**Comment by ghost** [comment]
specially the ui which he once said he hired a designer If I'm not mistaken and It's such a big source of inspiration for me to create web-applications in this type of ui which is awsome
